src/components/header/Header.css

```diff
@@ -277,10 +284,10 @@
   .header__button {
```

Comment:
Is there intentionally no transition here? Just noticed this. It's on mobile, so probably fine not to have a hover class at all anyways, but if there is one, we should probably have a transition.

Reply:
There's no need for a hover class on mobile. I've tested mobile apps with my S pen, which shows a cursor if I'm close enough to the Samsung Note screen, and I've yet to find any hover effects. And we can't detect hovering fingers, yet....